BUILD.md

```diff
@@ -144,6 +144,10 @@ bin/conduit stat deployments
 
 # view a live pipeline of requests
 bin/conduit tap deploy emojivoto/voting
+
+# view grafana dashboard
+kubectl -n conduit port-forward $(kubectl --namespace=conduit get po --selector=app=grafana -o jsonpath='{.items[*].metadata.name}') 3000:3000
```

Comment:
this will probably be replaced with a more formalized `conduit dashboard`-like command

Reply:
`conduit dashboard` already makes grafana available at `/api/v1/namespaces/conduit/services/grafana:http/proxy`.  The problem is that it seems like grafana doesn't use relative URLs so all links are broken.  Is it possible to configure grafana with an alternate root to make this work?

Reply:
Grafana can set an alternate root, though even without it this appears to just work on my system?

![screen shot 2018-02-28 at 10 58 01 am](https://user-images.githubusercontent.com/236915/36806911-63a5070a-1c76-11e8-8f74-f37469b12d6b.png)

A subsequent task in #420 is to determine the canonical way to access these dashboards, let's hold off until that change.